src/components/layouts/auth-wrapper/auth-wrapper.js

```diff
@@ -73,6 +75,15 @@ class AuthWrapper extends React.PureComponent {
     shouldLogout: false
   };
 
+  componentDidMount() {
+    let { token } = parse(this.props.location.search);
+    if (token) {
+      // eslint-disable-next-line no-undef
+      localStorage.setItem('token', token);
+      navigate('/');
```

Comment:
Navigate shouldn't redirect to /, it should redirect to the existing page without the token=xxxx part. 

> Note: Be carefull to not remove the rest of the query params (if there are any). For example if a users lands on `/member/1?edit=true&token=xxx` it should navigate to `/member/1?edit=true` . If a user lands on `/members?token=xxx` it should navigate to `/members`